.eslintrc

```diff
@@ -14,10 +15,11 @@
         "no-dupe-args": 2,
         "no-dupe-keys": 2,
         "no-duplicate-case": 2,
-        "no-empty-character-class": 2,
         "no-empty": 2,
+        "no-empty-character-class": 2,
         "no-ex-assign": 2,
         "no-extra-boolean-cast": 2,
+        "no-extra-parens": 0,
```

Comment:
This tunrs no.extra-parens, off, right? :+1: good, I like them most of the time for extra readability.